src/device-registry/utils/date.js

```diff
@@ -126,6 +126,16 @@ const addDays = (number) => {
   }
 };
 
+const addMinutes = (number) => {
+  try {
```

Comment:
New method for adding minutes to a date, although not used anywhere in the introduced functionality of this PR. Just here for future usage.